collection-scripts/gather_ceph_resources

```diff
@@ -388,42 +388,52 @@ for ns in $namespaces; do
-                for rns in $rados_namespaces; do
-                    {
-                        printf "Collecting mirror pool status in: %s/%s\n" "${bp}" "${rns}"
-                        timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool status --verbose $bp --namespace ${rns} --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-status-"${bp}"-debug.log
-                    } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_status_"${bp}"_"${rns}" &
-                    pids_rbd+=($!)
-                    {
-                        printf "Collecting mirror pool info in: %s/%s\n" "${bp}" "${rns}"
-                        timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool info $bp --namespace ${rns} --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-info-"${bp}"-debug.log
-                    } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_info_"${bp}"_"${rns}" &
-                    pids_rbd+=($!)
-                    images=$(timeout 60 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd ls -p $bp --namespace ${rns}")
-                    for image in $images; do
-                        dbglogf "${CEPH_GATHER_DBGLOG}" "Printing information for image: ${image}"
+                rados_namespaces=$(timeout 60 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd -p $bp namespace ls")
+                # skip the header from above output
+                namespaces="${rados_namespaces[*]:5}"
+                for rns in $namespaces; do
+                    isEnabled=$(timeout 60 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool info $bp --namespace ${rns} --format=json | jq --raw-output '.mode'")
+                    if [ "${isEnabled}" != "disabled" ]; then
+                        dbglogf "${CEPH_GATHER_DBGLOG}" "Mirroring is enabled on: ${bp} ${rns}"
                         {
-                            printf "Collecting mirror image status for images in: %s/%s/%s\n" "${bp}" "${rns}" "${image}"
-                            timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror image status $bp/$image --namespace ${rns} --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-image-status-"${bp}"-"${image}"-debug.log
-                        } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status_"${image}"_"${rns}".part &
+                            printf "Collecting mirror pool status in: %s/%s\n" "${bp}" "${rns}"
+                            timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool status --verbose $bp --namespace ${rns} --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-status-"${bp}"-"${rns}"-debug.log
+                        } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_status_"${bp}"_"${rns}" &
                         pids_rbd+=($!)
```

Comment:
Should `pids_rbd` array be cleared at some point before this line?  It seems like it would still contain PIDs from collecting data for the default namespace of the pool.